src/components/AddTodo/index.jsx

```diff
@@ -0,0 +1 @@
+export * from './AddTodo';
```

Comment:
Do you need similar files everywhere? Consider will be better if you will remove it 
<img width="439" alt="image" src="https://github.com/user-attachments/assets/6cced2cc-51ac-4229-b124-a6858135edd4" />